setup.cfg

```
@@ -43,6 +43,14 @@ install_requires =
     dataclasses; python_version < "3.7.0"
 python_requires = >=3.6.1
 
+[options.extras_require]
+ansible =
+    ansible
+ansible-base =
+    ansible-core
```

Oh, I missed this too
```suggestion
    ansible-base
```